.github/workflows/licenses.yaml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
*  verified that this works by adding an incompatible license and then running the check - flagged correctly. 
* moving this to its own hook makes golang no longer a requirement to run pre-commit 

```
go get github.com/alist-org/alist@3.28.0
```